source/beamline/BioXAS/BioXASMainBeamline.h

```diff
@@ -112,7 +113,7 @@ class BioXASMainBeamline : public BioXASBeamline
 	virtual BioXAS32ElementGeDetector* ge32DetectorOutboard() const { return 0; }
 
 	/// Returns the zebra control box.
-	virtual BioXASZebra *zebra() const { return zebra_; }
+	virtual BioXASMainZebra *zebra() const { return zebra_; }
```

Comment:
Or specifically in these subclasses you could have the convenience getters (if you were to do it in a separate issue).